packages/vscode/src/index.ts

```diff
@@ -57,7 +57,7 @@ export async function activate(context: ExtensionContext) {
 
 	// Restart the language server if any critical files that are outside our jurisdiction got changed (tsconfig, jsconfig etc)
 	workspace.onDidSaveTextDocument(async (doc: TextDocument) => {
-		const fileName = doc.fileName.split('/').pop() ?? doc.fileName;
+		const fileName = doc.fileName.split(/\/|\\/).pop() ?? doc.fileName;
```

Comment:
This is probably a good case to use path.sep: https://nodejs.org/api/path.html#pathsep

Reply:
Unfortunately, I believe `path` is not available for web extensions in this context:
> Node.js globals and libraries such as process, os, setImmediate, path, util, url are not available at runtime

Reply:
Ah, ok that make sense.